packages/flutter_tools/lib/src/version.dart

```diff
@@ -782,14 +805,14 @@ class GitTagVersion {
       z: parsedParts[2],
       devVersion: devParts[0],
       devPatch: devParts[1],
-      commits: parsedParts[4],
-      hash: parts[5],
-      gitTag: '${parts[0]}.${parts[1]}.${parts[2]}${parts[3] ?? ''}', // x.y.z-dev.m.n
+      commits: 0,
+      hash: '',
```

Comment:
Are these used anymore?

Reply:
they will only be present if we're parsing the output of `git describe`. this method is checking if we are exactly on a tag.